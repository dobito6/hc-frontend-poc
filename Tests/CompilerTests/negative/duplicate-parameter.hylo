fun f(x: Void, y: Void, x: Void) {}
