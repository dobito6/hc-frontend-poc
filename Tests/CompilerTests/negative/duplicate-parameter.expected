negative/duplicate-parameter.hylo:1.25-26: error: duplicate declaration
fun f(x: Void, y: Void, x: Void) {}
                        ^
negative/duplicate-parameter.hylo:1.7-8: note: previous declaration here
fun f(x: Void, y: Void, x: Void) {}
      ^
